### PR TITLE
Fix YAML syntax using yamllint.

### PR DESCRIPTION
**Pull request by zapashcanon**
My first PR. \o/

Will be self-merging this if no Travis error.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+---
```
Comment (K-Phoen) [inline .travis.yml]: Why adding this?

File: Resources/config/form.yml [hunk 2]
```diff
@@ -1,8 +1,9 @@
+---
 parameters:
     contact.form.type.message.class: KPhoen\ContactBundle\Form\Type\MessageType
 
 services:
     contact.form.type.message:
-        class: %contact.form.type.message.class%
+        class: '%contact.form.type.message.class%'
```
Comment (K-Phoen) [inline Resources/config/form.yml]: This change is useful, but the other ones I'm not sure :|

**Comment by K-Phoen** [comment]
What's the point of all these "whitespace" changes?

File: .travis.yml (hunk 1 above)
Comment (zapashcanon) [inline .travis.yml]: Otherwise:

    warning  missing document start "---"  (document-start)

**Comment by zapashcanon** [comment]
They are considered as errors by yamllint ; except the "---" document start which was just a warning.

File: .travis.yml (hunk 1 above)
Comment (K-Phoen) [inline .travis.yml]: I don't know what's best. Following the warnings triggered by your tool and write strictly valid YAML or just following how the Symfony community writes YAML.

**Comment by K-Phoen** [comment]
Not sure if this tool is relevant is our context ¯\_(ツ)_/¯

**Comment by zapashcanon** [comment]
Seems like Symfony is trying to get closer to the YAML spec, see [this blog post](https://symfony.com/blog/new-in-symfony-2-8-yaml-deprecations) ; but, I think you know what's best more than me. ;)

**Comment by K-Phoen** [comment]
> I think you know what's best more than me. ;)

No I don't ;)
You're the maintainer now, the final decision is up to you.

**Comment by zapashcanon** [comment]
I'll try to get more informations about the Symfony plans for the future and then, choose what to do.

**Comment by fabpot** [comment]
Just my 2cents here: whitespaces are not part of the YAML standard. It's up to you to determine what you want to do. Quoting is indeed important. Basically, if it is parseable by Symfony 3.1, it means that your YAML is reasonably "standard".

**Comment by zapashcanon** [comment]
Right @fabpot, thanks. I thought whitespaces were in the YAML standard as yamllint set them to error by default.

It's true some of the code is less readable now. But, yamllint allows the code to be more consistent and to throw some important errors - e.g. quoting related ones.

I'd like to keep it like this for now and override the default yamllint config to make it more readable later.

What do you think @K-Phoen ?

**Comment by zapashcanon** [comment]
I'm merging this.

**Pull request merged**